Sources/Apodini/Semantic Model Builder/InterfaceExporter/RequestHandler.swift

```diff
@@ -32,16 +33,33 @@ class InternalEndpointRequestHandler<I: InterfaceExporter, H: Handler> {
         return EventLoopFuture<Void>
             .whenAllSucceed(guardEventLoopFutures, on: request.eventLoop)
             .flatMapThrowing { _ in
-                try connection.enterConnectionContext(with: self.endpoint.handler) { handler in
+                try connection.enterConnectionContext(with: self.instance.handler) { handler in
                     try handler.handle()
                         .transformToResponse(on: request.eventLoop)
                 }
             }
-            .flatMap { typedAction -> EventLoopFuture<Response<AnyEncodable>> in
-                self.transformResponse(typedAction.typeErasured, using: connection, on: request.eventLoop, using: self.endpoint.responseTransformers)
+            .flatMap { typedAction -> EventLoopFuture<Response<HandledRequest>> in
+                let transformed = self.transformResponse(
+                    typedAction.typeErasured,
+                    using: connection,
+                    on: request.eventLoop,
+                    using: self.instance.responseTransformers)
```

Comment:
Minor: From a code formatting perspective I would move the parenthesis to the next line.
```suggestion
                let transformed = self.transformResponse(
                    typedAction.typeErasured,
                    using: connection,
                    on: request.eventLoop,
                    using: self.instance.responseTransformers
                )
```

Reply:
Solved via 5ac7d7df3c2c1de6c3fd59031fa9f758d6822218